examples/Building-and-deploying-multi-stage-RecSys/README.md

```diff
@@ -1,45 +1,66 @@
 # Deploying a Multi-Stage Recommender System
 
-We created two Jupyter notebooks that demonstrate two different stages of a Recommender Systems.
-The goal of the notebooks is to show how to deploy a multi-stage Recommender System and serve recommendations with Triton Inference Server.
+We created two Jupyter notebooks that demonstrate two different stages of recommender systems.
+The notebooks show how to deploy a multi-stage recommender system and serve recommendations with Triton Inference Server.
 The notebooks demonstrate how to use the NVTabular, Merlin Models, and Merlin Systems libraries for feature engineering, training, and then inference.
 
 The two example notebooks are structured as follows:
 
-- [Building the Recommender System](01-Building-Recommender-Systems-with-Merlin.ipynb): 
+- [Building the Recommender System](01-Building-Recommender-Systems-with-Merlin.ipynb):
   - Execute the preprocessing and feature engineering pipeline (ETL) with NVTabular on the GPU/CPU.
   - Train a ranking and retrieval model with TensorFlow based on the ETL output.
   - Export the saved models, user and item features, and item embeddings.
 
-- [Deploying the Recommender System with Triton](02-Deploying-multi-stage-RecSys-with-Merlin-Systems.ipynb): 
+- [Deploying the Recommender System with Triton](02-Deploying-multi-stage-RecSys-with-Merlin-Systems.ipynb):
   - Set up a Feast feature store for feature storing and a Faiss index for similarity search.
   - Build a multi-stage recommender system ensemble pipeline with Merlin Systems operators.
   - Perform inference with the Triton Inference Server using the Merlin Systems library.
 
 ## Running the Example Notebooks
 
-Merlin docker containers are available on http://ngc.nvidia.com/catalog/containers/ with pre-installed versions. For `Building-and-deploying-multi-stage-RecSys` example notebooks we used `merlin-tensorflow-inference` container that has NVTabular with TensorFlow and Triton Inference support.
+Containers with the Merlin libraries are available from the NVIDIA NGC catalog.
+To run the sample notebooks, use the `merlin-tensorflow` container.
+
+To run the example notebooks using a container, do the following:
+
+   You can use the `nvcr.io/nvidia/merlin/merlin-hugectr:nightly` container.
```

Comment:
Why should we use the merlin-hugectr nightly container? The two notebooks are based on tensorflow?

Reply:
Oh, I'm an idiot.  That's a typo.  Thank you.

Regarding `nightly`, last time I looked the nightly containers have to pass the same tests that the monthly release containers have to pass.  I'd love if we could restore using `latest` as a decent in-between, but I don't get to make that happen.